index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Main Css file -->
+    <link rel="stylesheet" href="css/master.css" />
+    <!-- Render all elements normally -->
+    <link rel="stylesheet" href="css/Normalize.css" />
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio Website: Mohammed Raad</title>
+  </head>
+  <body>
+    <!-- Toolbar start -->
+    <nav id="nav">
```

Comment:
- [ ] Nice work done on the header, but kindly move the nav and logo into a header tag for better use of semantic HTML